specifyweb/frontend/js_src/lib/components/Statistics/hooks.tsx

```diff
@@ -294,7 +246,7 @@ export function statsToTsv(
     statsText.itemName(),
     statsText.itemValue(),
   ];
-  const statItems = Object.entries(layout as IR<StatLayout>).flatMap(
+  const statItems = Object.entries(layout as IR<RA<StatLayout>>).flatMap(
```

Comment:
why do this type assertion? it's unsafe. Instead use `?? []` or `= []`

Also, this code seems overly complicated. why not just do `Object.entries(layout)[sourceIndex]`?